iosHyperskillApp/iosHyperskillApp/Sources/Views/SwiftUI/NavigationBar/StreakBarButtonItem.swift

```diff
@@ -10,10 +10,14 @@ struct StreakBarButtonItem: View {
             action: onTap,
             label: {
                 HStack {
-                    Image(Images.NavigationBar.streak)
-                        .renderingMode(.original)
-                        .resizable()
-                        .aspectRatio(contentMode: .fit)
+                    Image(
+                        currentStreak > 0
```

Comment:
We need to compare not the current streak sequence, but if completed today or not `state.streak?.history?.firstOrNull()?.isCompleted == true`